Update to xclim 0.61, xsdba 0.6.1 and xscen 0.15 (#537)
bird-house/finch: docs/source/installation.rst

   $ git clone https://github.com/bird-house/finch.git
   $ cd finch

> Create Conda environment named `finch`, this includes the development and documentation dependencies :

Reviewer: ```suggestion
Create Conda environment named `finch` (including the development and documentation dependencies):
```

   OR
   $ make install

Install from Conda
------------------

.. note::

>    `finch` is not yet available on conda-forge. We are not working on making this package available soon.

Reviewer: Are you sure about that? https://pypi.org/project/birdhouse-finch/

Author: Hmm, this is on pypi ? I don't think there's work in progress to put it on conda-forge ? There's also no need I think.

Reviewer: True.

```suggestion
   There are no plans to make `finch` available on conda-forge.
```
